OpenTabletDriver.Configurations/Configurations/ViewSonic/Woodpad PF1030.json

```
@@ -0,0 +1,35 @@
+{
+  "Name": "ViewSonic Woodpad PF1030",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 224,
+      "Height": 140,
+      "MaxX": 44800,
+      "MaxY": 28000
+    },
+    "Pen": {
+      "MaxPressure": 4095,
+      "ButtonCount": 2
+    },
+    "AuxiliaryButtons": null,
+    "MouseButtons": null,
+    "Touch": null
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 1347,
+      "ProductID": 58983,
```

I have seen 2 diagnostics from this tablet now and neither of them have 5898**3** as their PID, they have 5898**4** instead.

You seemed to have mistyped when you converted from hex in your original commit.
```suggestion
      "ProductID": 58984,
```